src/output/help.rs

```diff
@@ -51,15 +51,17 @@ pub(crate) struct Help<'help, 'app, 'parser, 'writer> {
 impl<'help, 'app, 'parser, 'writer> Help<'help, 'app, 'parser, 'writer> {
     const DEFAULT_TEMPLATE: &'static str = "\
         {before-help}{bin} {version}\n\
-        {author-with-newline}{about-with-newline}\n\
+        {author-section}\
+        {about-section}\n\
         {usage-heading}\n    {usage}\n\
         \n\
         {all-args}{after-help}\
     ";
 
     const DEFAULT_NO_ARGS_TEMPLATE: &'static str = "\
-        {before-help}{bin} {version}\n\
-        {author-with-newline}{about-with-newline}\n\
+        {before-help}{bin} {version-with-newline}\
```

Comment:
This is different from `DEFAULT_TEMPLATE ` above. What's the difference between `{version-with-newline}` and `{version}\n`? Why splitting them?

Reply:
`version-with-newline` only adds a newline when version exists? But I don't think that's what we want.

Reply:
whoopsie! These are some leftovers of my development phase. I remove them!

Reply:
I rebased and remove these leftovers, sorry for inconvenience.